heron/common/src/java/com/twitter/heron/common/basics/PackageType.java

```diff
@@ -0,0 +1,32 @@
+// Copyright 2016 Twitter. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.twitter.heron.common.basics;
+
+import com.google.common.io.Files;
+
+/***
+ * This enum defines commands topology package type
+ */
+public enum PackageType {
+  PEX,
+  JAR,
+  TAR;
+
+  public static PackageType getPackageType(String topologyBinaryFile) {
```

Comment:
how about `packageFile`, since this class is `PackageType`? `filename` is too general